### ISSUE TITLE
Extracting lower level features

### ISSUE THREAD

**Issue by wingz1** [metalbubble/TRN-pytorch#3]
Thanks for sharing this code.  It works great as advertised for testing. 
So, this is not an issue but a question.
What simple modification could be made (to test_video.py) to extract out some lower level features (before the "logits", say each of the vectors for each N-frame relation right after the g_theta MLP)???     (I've not coded in pytorch, so I have no idea where to start - now if it was Caffe, that's a different story.)    I'm curious how well such features might do in transfer learning for other videos not part of the training.  Or, do you think the final (akin to an fc8 layer) is the best for that?   You did a TSNE plot in your paper-- I assume those were from right after the h_phi MLP?)
Thanks!

**Comment by valeriechen** [comment]
I am also wondering about feature extraction, and was wondering which layer was used for the tSNE plots as well?
